sdk/head_tracker.h

```diff
@@ -42,11 +42,15 @@ class HeadTracker {
   // Resumes tracking ans sensors.
   void Resume();
 
-  // Gets the predicted pose for a given timestamp.
-  // TODO(b/135488467): Support different display to sensor orientations.
+  // Gets the predicted pose as a rotation for a given timestamp by forwarding a call 
```

Comment:
Consider not changing this comment header other than for adding the "timestamp == 0" special case.

Reply:
I left this on purpose. Let me know if still needs a change.

Reply:
Consider avoiding implementation details when writing a public API comment. The user of the class only needs to know what a certain methods does instead of how it does it (if it calls other methods, etc.).

Also, I think the "Gets the predicted pose as a rotation for a given timestamp" comment isn't accurate. It returns the pose (a rotation *and* a position).